ocd-3519.sql

```diff
@@ -0,0 +1,558 @@
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EC Group', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EC Group');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EC Individual (TIN/NPI)', -1
+WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EC Individual (TIN/NPI)');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EH/CAH Stage 3', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EH/CAH Stage 3');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EP Stage 2', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EP Stage 2');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EC ACI', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EC ACI');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EC ACI Transition', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EC ACI Transition');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EH/CAH Stage 2', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EH/CAH Stage 2');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'GAP-EH/CAH Stage 2', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'GAP-EH/CAH Stage 2');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'GAP-EH/CAH Stage 3', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'GAP-EH/CAH Stage 3');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'GAP-EP Stage 2', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'GAP-EP Stage 2');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'GAP-EP Stage 3', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'GAP-EP Stage 3');
+
+
+------------------- INSERT measure_typeS -------------------
+INSERT INTO openchpl.measure_type
+(name, last_modified_user)
+SELECT 'G1', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_type WHERE name = 'G1');
+
+INSERT INTO openchpl.measure_type
+(name, last_modified_user)
+SELECT 'G2', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_type WHERE name = 'G2');
+
+------------------- INSERT MIPS_MEASURES -------------------
+INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EC ACI Transition'), 'RT1', 'Electronic Prescribing: Eligible Clinician', 'Required Test 1: Promoting Interoperability Transition Objective 2 Measure 1 ', false, true, -1;
```

Comment:
The "required_test" here has a space at the end of it. Is that required?

Reply:
Ctrl-F and then ` ',` finds four others in the "insert MIPS measures" section

Oh, that comment could be updated too, though not required